standard/statements.md

```diff
@@ -1388,6 +1367,66 @@ finally
 }
 ```
 
+In the case where the expression `enumerable` represents a method call expression and one of the parameters is marked with the `EnumeratorCancellationAttribute` (§enumerator-cancellation) the a `CancellationToken` is passed to the `GetAsyncEnumerator` method. Other library methods may require a `CancellationToken` is passed to `GetAsyncEnumerator`. When those methods are part of the expression `enumerable`, the tokens shall be combined into a single token as if by `CreateLinkedTokenSource` and its `Token` property.
```

Comment:
Nit: `enumerable` could be a method call expression, but not specify an argument for a cancellation token due to the parameter being optional. I'd suggest that if we agree that *is* a problem, we address it in a separate PR.